sage/pallet/src/lib.rs

```diff
@@ -103,6 +103,41 @@ pub mod pallet {
 
 	pub type FungiblesAssetIdOf<T, I> = <T as Config<I>>::FungiblesAssetId;
 
+	#[pallet::genesis_config]
+	#[derive(frame_support::DefaultNoBound)]
+	pub struct GenesisConfig<T: Config<I>, I: 'static = ()> {
+		/// Genesis organizer account
+		pub organizer: Option<AccountIdOf<T>>,
+		/// Genesis initial season
+		pub season: Option<SeasonIdOf<T, I>>,
+	}
+
+	#[pallet::genesis_build]
+	impl<T: Config<I>, I: 'static> BuildGenesisConfig for GenesisConfig<T, I> {
+		fn build(&self) {
+			if let Some(ref organizer) = self.organizer {
+				Organizer::<T, I>::set(Some(organizer.clone()));
+
+				GeneralConfigStore::<T, I>::set(GeneralConfigOf::<T, I> {
+					transfer: TransferConfig { open: true },
+					trade: TradeConfig { open: true },
+					..Default::default()
+				});
+
+				if let Some(ref season_id) = self.season {
+					PlayerSeasonConfigs::<T, I>::insert(
+						organizer,
+						season_id,
+						PlayerConfig {
+							inventory_tier: InventoryTier::One,
+							locks: Locks::all_unlocked(),
+						},
+					);
+				}
+			}
```

Comment:
I have some doubts about implicitly assuming values here instead of explicitly setting values, but I guess you did this to avoid hassling with more trait bounds?